src/query/service/src/global_services.rs

```diff
@@ -146,6 +147,7 @@ impl GlobalServices {
             &config.query.max_server_memory_usage,
             config.query.tenant_id.tenant_name().to_string(),
         )?;
+        TempDirManager::init(&config.spill, config.query.tenant_id.tenant_name())?;
```

Comment:
Do not merge, TempDirManager does not work on cloud

Reply:
`pr-16441-a41f29e-1727715020` is ready for cloud